docs/userguide/troubleshooting.rst

```diff
@@ -18,19 +19,37 @@ Excel/Word or another MS Office software says the file name is too long
 
 So this is a Microsoft issue, and there is nothing Parsec can do about it. The only solution is to rename the file, or rename one or more folders that contain the file in order to have shorter names.
 
-Too many old devices appear in the device selection dropdown
-------------------------------------------------------------
+Specific to Parsec V2
+---------------------
+
+Copying files from Parsec V2 take some time / files are not accessible during copy
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+When copying files from a mounted workspace in Parsec V2 the operation may take some time and the files may not be accessible to open them. After some time, files are copied and can be open normally.
+
+This is most likely due to data is not being present locally in cache. Reading data when is not in cache is much slower than reading data from from a regular on-disk filesystem and performance is heavily dependent on your internet connection speed.
+
+Moreover, while the file is under copy, its displayed size only grows when a *flush* operation occurs. Flush are not explicitly issued by Parsec V2 and happen automatically when data written since previous flush is bigger than the blocksize of the file (i.e. 512KB).
+
+So this explain why the file can appear with a size of 0 for some time: data is being downloaded into cache and then copied via occasional flush operations.
+
+You could try enabling :ref:`off-line mode <doc_userguide_workspace_offline_mode>` on workspaces before copying large amount of files. In any case, you will need to wait for the copy to be fully finished to be able to open files.
+
+How to remove old devices from the login page
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Old devices remain registered on the computer and are displayed in the login page. This is usually the case after having tried multiple Parsec beta versions.
 
-This usually occurs when a user has tried Parsec beta versions in which test databases could be reset at any time, leaving old devices still registered on their computer.
+You can still remove these devices manually:
 
-A user can still erase these devices manually though:
+1. Go to Parsec configuration directory
 
-**On Windows**, in the explorer's address bar, go to ``%APPDATA%/parsec**`` (or ``C:\Users\<YourUser>\AppData\Roaming``, AppData is a hidden folder).
-**On Linux**, go to ``~/.config/parsec``.
+   - **On Windows**, open the file explorer and put ``%APPDATA%/parsec**`` in the address bar (or go to ``C:\Users\<YourUser>\AppData\Roaming``, note that AppData is a hidden folder).
+   - **On Linux**, open ``~/.config/parsec``.
 
-- Remove unwanted devices in config/devices
-- Remove unwanted devices in data
+2. Remove unwanted devices in config/devices
```

Comment:
The path has changed since #7527.

Should be `libparsec/config/devices` :thinking:

Reply:
Notice this is inside a section "Specific to Parsec V2" that I've just added. 

Besides, I'm not sure if that is still an issue in V3 since the concept of devices is more or less hidden to the user and I think only the more recent device is displayed.